dev.exs

```diff
@@ -142,14 +149,6 @@ seeds = fn ->
     content: "body {cursor: zoom-in;}"
   })
 
-  Beacon.Content.create_component!(%{
-    site: "dev",
-    name: "sample_component",
-    body: """
-    <%= @val %>
-    """
-  })
```

Comment:
Please revert this change, this component is used on the home page for local development.

Reply:
Done